src/utils/winston.js

```diff
@@ -1,16 +1,19 @@
 import { createLogger, format, transports } from 'winston';
 
 const {
-  combine, timestamp, simple, json
+  combine, timestamp, simple, json, prettyPrint, colorize
```

Comment:
The `colorize` format seems to be messing up the log files.

Reply:
Yeah yea, I have noticed now that the log files. Thanks